org/issue.ts

```diff
@@ -4,27 +4,89 @@ const gh = danger.github as any;
 const issue = gh.issue;
 const repo = gh.repository;
 
-const qWords: string[] = ["who", "what", "where", "when", "why", "how"]
+const title = issue.title.toLowerCase()
+
+var labelsToAdd: string[] = []
+
+// label: question
+
+const qWords: string[] = ["how", "who", "what", "where", "when", "why", "which"]
 
 if (issue.title.slice(-1) == "?") {
-  addLabel()
+  labelsToAdd.push("question")
 } else {
-  const title = issue.title.toLowerCase()
   for (var i = 0; i < qWords.length; i++) {
     if (title.startsWith(qWords[i])) {
-      addLabel()
+      labelsToAdd.push("question")
+      break
+    }
+  }
+}
+
+// label: documentation
+
+const docWords: string[] = ["documentation", "document", "docs", "readme", "changelog", ".md"]
+
+for (var i = 1; i < docWords.length; i++) {
+  if (title.includes(docWords[i])) {
+    labelsToAdd.push("documentation")
+    break
+  }
+}
+
+// label: enhancement
+
+const enhanceWords: string[] = ["add", "allow", "improve", "upgrade", "update", "support"]
```

Comment:
I'm guessing `includes` would trigger some false positives. May be good to suffix keywords with a space?

Reply:
Or split the title based on spaces and look for the match?

Reply:
Yeah that's a much better way of doing it 👍 Forgot how small the string is